t/02-bowels/20-add.t

```diff
@@ -204,5 +204,15 @@ my $archive = make_dist_archive("$dist=$pkg1;$pkg2");
 }
 
 #-----------------------------------------------------------------------------
+subtest 'Allow dry run add on locked repo' => sub {
+
+    my $t = Pinto::Tester->new;
+    $t->run_ok( 'Lock' => {} );
+    $t->stack_is_locked_ok('master');
+    $t->run_ok( 'Add', { archives => $archive, dry_run => 1 } );
+    $t->registration_not_ok("AUTHOR/$dist/$pkg1/master");
```

Comment:
Asserting that something is **not** registered is a bit sketchy, since it is easy to get a false positive if you happen to mistype something in the test. It might be more robust to use `repository_clean_ok` to assert that the repo is still in a virgin state after the operation. What do you think?